casper/src/main/scala/coop/rchain/casper/api/BlockAPI.scala

```diff
@@ -810,7 +810,9 @@ object BlockAPI {
   }
 
   def exploratoryDeploy[F[_]: Monad: EngineCell: Log: SafetyOracle: BlockStore](
-      term: String
+      term: String,
+      blockHash: String,
+      usePreStateHash: Boolean
```

Comment:
@zsluedem why not set here default value `usePreStateHash: Boolean=false`? So this can be called as 
`BlockAPI.exploratoryDeploy(term, hash)`, and `BlockAPI.exploratoryDeploy(term, hash, true)` only when it is needed (I assume rarely)

Reply:
makes sense